### ISSUE TITLE
Error syntax print command in hacktronian.py

### ISSUE THREAD

**Issue by NetiSHTer22** [thehackingsage/hacktronian#34]
File "/usr/share/doc/hacktronian/hacktronian.py", line 82
    print """
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(""")?

Check this...

**Comment by risyadfahlevi** [comment]
im also to same error
  File "/usr/share/doc/hacktronian/hacktronian.py", line 82
    print """
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(""")?

**Comment by Hustleboibam** [comment]
Yes

ALOMO INC.

**Comment by SecurityJos** [comment]
solved it with running the older python version on the kali server if you run version 2.7.18 it works correctly
https://www.kali.org/docs/general-use/using-eol-python-versions/#pyenv

**Comment by MrAppAndCrap** [comment]
> solved it with running the older python version on the kali server if you run version 2.7.18 it works correctly https://www.kali.org/docs/general-use/using-eol-python-versions/#pyenv

That doesn't seem to be working for me.

**Comment by SecurityJos** [comment]
> 

what is the issue you run into on this moment is it the same with the print issue? 

did you check the version of python you are running on this moment when you try to start it? 
![image](https://user-images.githubusercontent.com/121550648/210723303-83d05abe-f13d-4f04-b717-e04fac729cde.png)

**Comment by punkrock4dadz** [comment]
If you have several versions of Python installed, /usr/bin/env will ensure the interpreter used is the first one on your environment's $PATH. The alternative would be to hard code something like #!/usr/bin/python.

In Unix, an executable file that's meant to be interpreted can indicate what interpreter to use by having a #! at the start of the first line, followed by the interpreter (and any flags it may need).

> _LINE ONE OF hacktronian.py:_

**#!/usr/bin/env python2.7.18**

In computing, a shebang (also called a hashbang, hashpling, pound bang, or crunchbang) refers to the characters "#!" when they are the first two characters in an interpreter directive as the first line of a text file. In a Unix-like operating system, the program loader takes the presence of these two characters as an indication that the file is a script, and tries to execute that script using the interpreter specified by the rest of the first line in the file.

In summary, this needs to be executed by python 2.7.18 because of indents and unindented blocks.
